cabal.project

```diff
@@ -13,26 +13,31 @@ source-repository-package
   type: git
   location: https://github.com/input-output-hk/cardano-base
   tag: 965b8b143632faea16680a195e6de57091382700
+  --sha256: 0x3cy5xa9mqdqnavs45assmmcrzw07qcwsv95capqwa6awz1plhh
   subdir: binary
 
 source-repository-package
   type: git
   location: https://github.com/input-output-hk/cardano-base
   tag: 965b8b143632faea16680a195e6de57091382700
+  --sha256: 0x3cy5xa9mqdqnavs45assmmcrzw07qcwsv95capqwa6awz1plhh
```

Comment:
Ditto, are these intended to be fully removed? Otherwise what does this indicate?

Reply:
cabal does not yet support checksum for source-repository-package, but nix need them, so the `--sha256` comment is used by haskell.nix for this purpose.
(this PR is aiming to remove all generated nix but the counter-part is that `cabal.project` need to provide the information that was previously generated in `nix/.stack.nix`)

Reply:
`/nix/regenerate.sh` can keep those sha256 in sync.